### PR TITLE
Added Heidelberg #1868

### PR DESCRIPTION
**Pull request by Vishrut19**
Added Heidelberg #1868

### CODE REVIEW

**Review by BaseMax — APPROVED** [review]
good

**Comment by BaseMax** [comment]
It seems something is wrong. John will explain to you when he can.

**Comment by Vishrut19** [comment]
okay but if @jbampton can explain then it is good for me as I can try to work on it and get the pr merged soon

File: doc/source/index.rst [hunk 1]
```diff
@@ -229,7 +230,7 @@ Welcome to slurp's documentation!
    aruba.md
    curacao.md
 
-.. toctree::
+ .. toctree::
```
Comment (jbampton) [inline doc/source/index.rst:233]:
```suggestion
.. toctree::
```

**Review by jbampton — APPROVED** [review]

**Pull request merged**